backend/src/main/java/codezap/category/domain/Category.java

```diff
@@ -50,18 +51,23 @@ public class Category extends BaseTimeEntity {
     @Column(nullable = false)
     private Boolean isDefault;
 
-    public Category(String name, Member member) {
+    @Column(nullable = false)
+    private long ordinal;
+
+    public Category(String name, Member member, long ordinal) {
         this.name = name;
         this.member = member;
         this.isDefault = false;
+        this.ordinal = ordinal;
     }
```

Comment:
`@AllArgsConstructor` 생성자 체이닝 제안해요~

```suggestion
    public Category(String name, Member member, long ordinal) {
        this(null, member, name, false, ordinal);
    }
```